src/components/organisms/PostNav/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  PostNavWrapper,
+  WorkspaceWrapper,
+  NavWrapper,
+  NavTitle,
+  NavContentWrapper,
+  NavContent,
+} from './index.styles';
+import { Text } from '../../atoms/Text';
+import { Button } from '../../atoms/Button';
+import { InternalLink } from '../../../routes/InternalLink';
+import { usePost } from '../../../context/Post/hook';
+import { useInternalRouter } from '../../../routes/internalRouter';
+
+export const PostNav = () => {
+  const { posts } = usePost();
+  const { push } = useInternalRouter();
+
+  return (
+    <PostNavWrapper>
+      <WorkspaceWrapper>
+        <Text rem={2} weight={400}>
+          My Workspace
+        </Text>
+      </WorkspaceWrapper>
+
+      <NavWrapper>
+        <NavTitle>
+          <Text rem={1.5} weight={400}>
+            공고 리스트
+          </Text>
+          <Button size={'fit'} onClick={() => push('/create')}>
+            +
+          </Button>
```

Comment:
이렇게 작성해도 충분하다고 생각하지만 더 라우팅 전용으로 컴포넌트를 만들어도 좋을거같아요.
```suggestion
          <RouteButton navigateRoute='/create'>
              <Button size={'fit'}>
                +
              </Button>
          <RouteButton />
```

얼마전에 [미션](https://github.com/EungyuCho/react-shopping-cart/blob/eungyucho-step1/client/src/components/Button/RouteButton.tsx)을 하다가 사용한 방법인데 보다 직관적으로 라우팅할 수 있는거같아서 코멘트 드려요 🙂